packages/modal/src/ModalPortal.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import Blanket from '@xcritical/blanket';
+import Portal from '@xcritical/portal';
+
+import {
+  ModalHeaderWrapper,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalIconClose,
+} from './styled';
+
+import { IModalProps } from './interfaces';
+import { getModalBlanketTheme } from './utils';
+import { IconClose } from './IconClose';
+
+
+export const ModalPortal: React.FC<IModalProps> = ({
```

Comment:
I am not sure about ModalPortal that almost duplicate Modal component structure.
May be it would be better expand Modal component logic with isInPortal prop?

Reply:
No, because Modal component includes logic with react context and providing zIndex to styled components, ModalPortal does not work with these things.